packages/bierzo-wallet/package.json

```diff
@@ -37,7 +37,7 @@
   "scripts": {
     "lint": "tsc --noEmit && eslint -c .eslintrc.js --max-warnings 0 'src/**/*.ts{,x}'",
     "format": "prettier --write --loglevel warn './src/**/*.{ts,tsx,json,md,css}'",
-    "start-developmen": "REACT_APP_CONFIG=development react-scripts start",
```

Comment:
Sorry about this one. Is fixed on master now

Reply:
This stuff happens, do not worry 😉